_static/custom.css

```
@@ -0,0 +1,3 @@
+html[data-accent-color='lime'] {
+    --accent-9: #7efa01;
+}
```

missing newline at EOF

```suggestion
}

```